bin/jsdoc-to-ts.js

```diff
@@ -4,19 +4,22 @@
 
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-const minimist = require('minimist');
-const jsdocToTs = require('..');
+import fs from 'fs';
+import url from 'url';
```

Comment:
Why don't we destruct fileURLtoPath from url like

import {fileURLToPath} from 'url';